portality/scripts/scrip_tags_check.py

```diff
@@ -44,10 +41,23 @@ def find_nested(substr, d, result):
         writer = csv.writer(f)
         writer.writerow(["ID", "Article Title", "field", "value"])
 
-        for j in esprit.tasks.scroll(conn, ipt_prefix(models.Article.__type__), q=ALL_ARTICLES, page_size=100, keepalive='5m'):
+        for j in esprit.tasks.scroll(conn, ipt_prefix(models.Article.__type__), q=ALL, page_size=100, keepalive='5m'):
             article = models.Article(_source=j)
-            print("Analyzing article: " + article.id)
             results = find_nested("<script>", article.__dict__, [])
 
             for key, value in results:
                 writer.writerow([article.id, article.bibjson().title, key, value])
+
+        for j in esprit.tasks.scroll(conn, ipt_prefix(models.Application.__type__), q=ALL, page_size=100, keepalive='5m'):
+            app = models.Application(_source=j)
+            results = find_nested("<script>", app.__dict__, [])
+
+            for key, value in results:
+                writer.writerow([app.id, app.bibjson().title, key, value])
```

Comment:
Might need to add 'application', to each row so that we know which IDs are for which type, since they are stored in different indexes.

Reply:
Done